### PR TITLE
🔒jh-MAS-72-lets-encrypt-certs

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

**Comment by nickmcmahon01** [comment]
Where were these copied from, why are they needed etc.

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ image:
 ingress:
   enabled: true
   enable_whitelist: true
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    certmanager.k8s.io/cluster-issuer: letsencrypt-production
+    nginx.ingress.kubernetes.io/limit-rpm: "60"
```
Comment (nickmcmahon01) [inline helm_deploy/values-dev.yaml:18]: We only have one client, how many requests per minute do we expect from Oasys and how does it cope with back pressure?

Comment (jonhindle) [reply helm_deploy/values-dev.yaml:18]: Yeah I lifted from offender-assessments-api-kotlin. I guess we don't need the rate line? I'll remove for now 👍

**Comment by astroash** [comment]
closed as project has changes teams and work is out of date with main

**Pull request closed without merging**